### DISCUSSION TITLE
Add High DPI Scaling with 125% and 150%

### DISCUSSION THREAD

**Discussion by Momp1** [AlexandreRouma/SDRPlusPlus#1078] [Ideas]
I have a 2560x1600 display and with 100% scaling, the GUI appears to small but with 200% it is way too big and significantly takes away space for the waterfall and spectrum to be displayed.

**Comment by sizzlik** [comment]
Same..150% would be nice. Or 25% steps. But it has been requested before and i think there was a reason why its not implemented (yet)

**Reply by Momp1** [reply]
ah alright I didn't see that. Thank you.
